lib/ui/window/pointer_data.h

```diff
@@ -9,6 +9,29 @@
 
 namespace flutter {
 
+// Must match the button constants in events.dart.
+namespace PointerButton {
+const int64_t kPrimary = 1 << 0;
```

Comment:
If you're not using enum, you need to put the definitions in a cc file and declare them extern here (until we have C++17's `inline constexpr`).

You could use `enum` (not `enum class`) as we do in embedder.h to avoid the need for that (I think it'll implicitly convert to int64_t despite being smaller intrinsically, in which case it wouldn't have impacts on usage).

Reply:
I fixed it using `enum`.